docker/cluster-test/cluster-test.Dockerfile

```diff
@@ -18,8 +18,8 @@ RUN rustup install $(cat rust-toolchain)
 COPY . /libra
 RUN docker/cluster-test/compile.sh
 FROM debian:buster
-RUN apt-get update && apt-get install -y openssh-client curl
-RUN curl -L "https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
+RUN apt-get update && apt-get install -y openssh-client curl wget
```

Comment:
remove curl?

Reply:
I don't know if Cluster test runner needs it.  Will defer to @andll

Reply:
Actually, we can find this out from LBT.  Will do that.